src/qml/SettingsPage.qml

```diff
@@ -28,6 +28,20 @@ Kirigami.ScrollablePage {
     ColumnLayout {
         width: parent.width;
 
+        Kirigami.AbstractCard {
+            Layout.fillWidth: true
+
+            header: ColumnLayout {
+                Kirigami.Heading {
+                    text: qsTr("Please read the instructions:");
```

Comment:
Hehe, this reads a whole lot like "rtfm", but we'd perhaps best let MasterTailer decide on the wording :D